### PR TITLE
Structure_Engine: Clean up ToBeRemoved methods

### LINKED ISSUE #2736: Structure_Engine: Cull all methods with Replaced or ToBeRemoved attributes
#### Broken rules:

The Structure_Engine contains a set of methods with ToBeRemoved as well as Replaced methods, generally with those arguments put there before the Versioning system was in place. 

#### Suggestions to restore compliance:

Version all methods that can be versioned that contain those arguments, and remove the rest that are past expiry date of at least 4 versions.

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### Issues addressed by this PR

Closes #2736

### CODE REVIEW

**Comment by IsakNaslundBh** [comment]
@BHoMBot check installer

Running to check more toolkits affected by the change

**Comment by IsakNaslundBh** [comment]
@BHoMBot check installer

Running to check more toolkits affected by the change

**Comment by IsakNaslundBh** [comment]
@BHoMBot check installer

**Comment by IsakNaslundBh** [comment]
@BHoMBot check versioning

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Comment by IsakNaslundBh** [comment]
@BHoMBot check required

File: Structure_Engine/Query/Normal.cs [hunk 1]
```diff
@@ -119,15 +119,16 @@ private static Vector Normal(this IAreaElement areaElement)
...
-        /**** Public Methods - ToBeRemoved               ****/
+        /**** Private Methods                           ****/
         /***************************************************/
 
         [ToBeRemoved("3.1", "ToBeRemoved by method targeting IElement2D.")]
         [Description("Returns the Panels local z-axis, a vector orthogonal to the plane of the Panel. This is found by fitting a plane through all the edge curves and taking the Normal from this plane.")]
         [Input("panel", "The Panel to evaluate the normal of.")]
         [Output("normal", "Vector representing the local z-axis Panel.")]
-        public static Vector Normal(this Panel panel)
+        private static Vector Normal(this Panel panel)
```
Comment (FraserGreenroyd) [inline Structure_Engine/Query/Normal.cs:129]: I note the method has been made private but the `ToBeRemoved` attribute still exists, might be worth removing the attribute at some point @IsakNaslundBh ? (Maybe another, smaller PR that doesn't impact 10 others with checks?)

**Review by FraserGreenroyd — COMMENTED** [review]
Seems there's a few `ToBeRemoved` attributes that may no longer be needed?

File: Structure_Engine/Query/Normal.cs (hunk 1 above)
Comment (IsakNaslundBh) [reply Structure_Engine/Query/Normal.cs:129]: ah, good catch! Think I just fix it in this PR for it not to be hanging again. Especially now that most alignment PRs already have been merged.

**Comment by FraserGreenroyd** [comment]
@IsakNaslundBh what's the sitrep on this PR then? 😄

**Comment by IsakNaslundBh** [comment]
@BHoMBot check versioning

**Comment by IsakNaslundBh** [comment]
@BHoMBot check versioning

**Comment by IsakNaslundBh** [comment]
@BHoMBot check compliance

**Comment by IsakNaslundBh** [comment]
@BHoMBot check core
@BHoMBot check null-handling
@BHoMBot check serialisation
@BHoMBot check unit-tests

**Comment by IsakNaslundBh** [comment]
I Have reviewed the failing UnitTest, which is due to changes of now defaulting orientation of nodes to null rather than world XY.

This change can be deemed fine, why the UnitTest safely can be updated. WIll do this and push to this PR.

**Comment by IsakNaslundBh** [comment]
@BHoMBot check unit-tests

**Review by peterjamesnugent — APPROVED** [review]
Checked through all the files changed and couldn't see anything that would cause an issue.

I have tested a few of the dependent PRs and they have worked without issue - will rely on the CI/CD checks to confirm any outstanding issues.

**Comment by IsakNaslundBh** [comment]
@BHoMBot check required

**Comment by IsakNaslundBh** [comment]
@BHoMBot check copyright-compliance
@BHoMBot check dataset-compliance

**Comment by IsakNaslundBh** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**